engine-standalone-storage/src/lib.rs

```diff
@@ -227,7 +227,7 @@ impl Storage {
     }
 
     fn process_transaction<F: Fn(&mut rocksdb::WriteBatch, &[u8], &[u8])>(
-        &mut self,
+        &self,
```

Comment:
I know that technically this doesn't mutate `self`, but it does internally because of the `self.db.write` call at the end of the function. I recommend adding an `allow` for whatever clippy lint is given an error on this function.